docs/pages/machine-id/getting-started.mdx

```diff
@@ -0,0 +1,120 @@
+---
+title: Machine ID Getting Started Guide
+description: Getting started with Teleport Machine ID
+---
+
+# Getting Started
+
+In this getting started guide we will use Machine ID to create an identity for a machine and use that identity to connect to a SSH server.
+
+Here's an overview of what we will do:
+
+1. Download and install Teleport (=teleport.version=).
```

Comment:
So it's clear how this guide applies to all Teleport deployments (including Cloud), I think this should indicate where in your deployment Teleport will be installed to follow this guide. For example:
- A new Teleport node for your existing cluster
- The Auth/Proxy service (i.e., this wouldn't be compatible with Cloud)
- Every host you plan to run the `teleport` daemon on, including the Auth Service, Proxy Service, and Nodes